jgrapht-core/src/main/java/org/jgrapht/Graphs.java

```diff
@@ -42,7 +42,9 @@
 package org.jgrapht;
 
 import java.util.*;
+import java.util.function.Predicate;
 
+import org.jgrapht.experimental.dag.DirectedAcyclicGraph;
```

Comment:
I moved the DAG-related methods to the DAG class itself (commit _9bd26b9c2ed9361dc291264a4fe2ee333fcccb1a_). There is no need for the new methods to be static anymore. Therefore I would like to convert the new methods in the DAG class to instance methods. What do you think?